proxy/upstreammode.go

```diff
@@ -23,6 +23,9 @@ const (
 	// or AAAA requests only with the fastest IP address detected by ICMP
 	// response time or TCP connection time.
 	UpstreamModeFastestAddr UpstreamMode = "fastest_addr"
+
+    // UpstreamModeRandom makes server to a random upstream
```

Comment:
Please, consider using `gofumpt` to format the code in this repository:
1. ```sh
   make go-tools
   ```
1. ```sh
   ./bin/gofumpt --extra -w .
   ```